include/hx/Object.h

```diff
@@ -166,11 +166,11 @@ class HXCPP_EXTERN_CLASS_ATTRIBUTES Object
          if (!alloc)
             BadImmixAlloc();
          #endif
-
+         
          return ImmixAllocator::alloc(alloc, inSize, inContainer, inName);
 
       #else // Not HX_USE_INLINE_IMMIX_OPERATOR_NEW ...
-
+            
```

Comment:
This file isn't actually changed …

```suggestion

```

Reply:
Yes sorry, i made cleanup for Immix.cpp but didn't check the include files (didn't sleep this night = P)